Sources/Seatsio/Pricing.swift

```diff
@@ -21,11 +21,7 @@ public struct Pricing: Codable {
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         if let value = try? container.decodeIfPresent(Int.self, forKey: .category) {
-            if (value == nil) {
-                category = nil
-            } else {
-                category = String(value!)
-            }
+            category = String(value)
```

Comment:
Is the if statement not needed anymore? Won't the code crash if no category is specified?

Reply:
In Swift 5, the value is no longer optional, which eliminates the risk of a crash since there’s no need for forced unwrapping. In earlier versions, however, it’s represented as a double optional (??), requiring two levels of unwrapping.